logic/atrium-logic-jvm/src/generated/kotlin/ch/tutteli/atrium/logic/path.kt

```diff
@@ -24,8 +26,8 @@ fun <T : Path> AssertionContainer<T>.startsNotWith(expected: Path): Assertion =
 fun <T : Path> AssertionContainer<T>.endsWith(expected: Path): Assertion = impl.endsWith(this, expected)
 fun <T : Path> AssertionContainer<T>.endsNotWith(expected: Path): Assertion = impl.endsNotWith(this, expected)
 
-fun <T : Path> AssertionContainer<T>.exists(): Assertion = impl.exists(this)
-fun <T : Path> AssertionContainer<T>.existsNot(): Assertion = impl.existsNot(this)
+fun <T : Path> AssertionContainer<T>.exists(vararg linkOptions: LinkOption): Assertion = impl.exists(this, *linkOptions)
```

Comment:
we never used varag on the logic level. Unless required to call the jvm API I would use List instead.

Reply:
Done